api-flask/docker-compose.test.yml

```diff
@@ -0,0 +1,23 @@
+version: '3'
+
+services:
+  api:
+    environment:
+      - 'RUN=uvicorn app.main:app --host 0.0.0.0 --port 80'
+      - POSTGRES_HOST=testdb
+      - POSTGRES_USER=postgres
+      - POSTGRES_DB=postgres
+      - POSTGRES_PORT=5432
+      - POSTGRES_PASSWORD=!ChangeMe!
+    command: uvicorn app.main:app --host 0.0.0.0 --port 80 --reload
+    # Infinite loop, to keep it alive, for debugging
+    # command: bash -c "while true; do echo 'sleeping...' && sleep 10; done"
+    #
+# run a full battery of test on all endpoints using https://pypi.org/project/schemathesis/
```

Comment:
should this be at the top?

Reply:
I'm not even sure why I put it there in the first place, to be honest :thinking: but yes, top of the file with some extra explanations. Added.